content/docs/1.4.2/deploy/upgrade/auto-upgrade-engine.md

```diff
@@ -38,8 +38,8 @@ So, Longhorn leaves it to you to decide when it is the good time to manually upg
 At this time, Longhorn will automatically do offline upgrade for the volume similar to the detached volume case.
 
 #### 3. What Happened If The Upgrade Fails?
-If a volume failed to upgrade its engine, the engine image in volume's spec will remain to be different than the engine image in the volume's status.
-Longhorn will continuously retry to upgrade until it succeeds.
+If a volume's engine fails to upgrade, the volume's status won't change. 
+Because the status will remain different from the spec, Longhorn will retry the upgrade until it succeeds.
```

Comment:
```suggestion
Because the status of the engine image will remain different from the spec, Longhorn will retry the upgrade until it succeeds.
```

Reply:
The issue with the original is that it leaves much implicit rather than explicit.  Customers were confused.
How do I fix other versions?